datastream-kafka-connector/src/test/java/com/linkedin/datastream/connectors/kafka/mirrormaker/TestKafkaMirrorMakerConnectorTask.java

```diff
@@ -1184,8 +1221,6 @@ public void testInFlightMessageCount() throws Exception {
 
     // verify the states response returned by diagnostics endpoint contains correct counts
     KafkaDatastreamStatesResponse statesResponse = connectorTask.getKafkaDatastreamStatesResponse();
-    Assert.assertEquals(statesResponse.getAutoPausedPartitions().size(), 3,
-        "All topics should have had auto-paused partitions");
     Assert.assertEquals(
         statesResponse.getInFlightMessageCounts().get(new FlushlessEventProducerHandler.SourcePartition(yummyTopic, 0)),
```

Comment:
While you're at it, can you also fix these logs? They all print "yummyTopic" even though we check for the other topics.